mir-webapp/pom.xml

```diff
@@ -89,12 +90,14 @@
       <plugin>
         <groupId>org.mycore.plugins</groupId>
         <artifactId>solr-runner-maven-plugin</artifactId>
-        <version>1.0</version>
+        <version>1.1-SNAPSHOT</version>
         <configuration>
           <solrHomeTemplate>src/main/solr</solrHomeTemplate>
           <solrHome>${mir.solr.home}</solrHome>
           <solrMirrorURL>${solr-runner.solrMirrorURL}</solrMirrorURL>
           <solrVersionString>${solr-runner.solrVersionString}</solrVersionString>
+          <gitRepository>https://github.com/MyCoRe-Org/mycore_solr_configset_main.git</gitRepository>
```

Comment:
we should stick to the mir-solr submodule. please make a branch there with the configsets (further submodules?) and use this branch here. There is a need to support at least 2 different config sets: "mir" and "classification". Sticking to the submodule does not require a new release of the solr-runner-maven-plugin.